### ISSUE TITLE
Cache is not working

### ISSUE THREAD

**Issue by jonathborg** [strapi-community/strapi-plugin-rest-cache#3]
I have followed the docs. I just execute with DEBUG=strapi:* yarn develop, but I dont have headers or logs about caching.

config/plugins.js
```
module.exports = {
  "strapi-plugin-rest-cache": {
    config: {
      provider: {
        name: "memory",
        options: {
          max: 32767,
          maxAge: 3600,
        },
      },
      strategy: {
        enableXCacheHeaders: true,
        contentTypes: [
          "api::restaurant.restaurant",
        ],
      },
    },
  },
};

```

**Comment by eveyrat** [comment]
@jonathborg I'm facing the same problem, looks like NPM packages are empty.

I've followed the docs for installing:

```
yarn add \
  strapi-plugin-rest-cache \
  strapi-plugin-redis \
  strapi-provider-rest-cache-redis
```

```
# package.json

"dependencies": {
    ....
    ....
    "strapi-plugin-redis": "^1.0.0-alpha.1", <= installed correctly
    "strapi-plugin-rest-cache":"^4.0.0",
    "strapi-provider-rest-cache-redis":"^4.0.0"
  },
```
  
  
![image](https://user-images.githubusercontent.com/403226/153856985-c86325fc-7500-4661-9da1-9306e9229a62.png)

  
I could only make it work by using local packages.

BTW, awesome plugin,  thanks guys.

**Comment by jerson-dp** [comment]
> eveyrat

I am facing the same issue..Followed as per the doc, but caching to redis is not working and not finding any logs trace on this...May I know how you managed to get it work?

**Comment by stafyniaksacha** [comment]
Hello!

The issue is solved on the `4.0.2`! 

Also note that the plugin configuration should be `rest-cache` instead of `strapi-plugin-rest-cache`
